test/e2e/set/scylladbmanagertask/multidatacenter/scylladbmanagertask_scylladbcluster_globalmanager_object_storage.go

```diff
@@ -0,0 +1,556 @@
+		defer targetSCRolloutAfterForcedRedeploymentCtxCancel()
+		targetSC, err = controllerhelpers.WaitForScyllaDBClusterState(targetSCRolloutAfterForcedRedeploymentCtx, f.ScyllaAdminClient().ScyllaV1alpha1().ScyllaDBClusters(targetSC.Namespace), targetSC.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaDBClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		scylladbclusterverification.Verify(ctx, targetSC, rkcClusterMap)
+		err = scylladbclusterverification.WaitForFullQuorum(ctx, rkcClusterMap, targetSC)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Creating a tables restore task against global ScyllaDB Manager instance")
+		tablesRestoreCreationCtx, tablesRestoreCreationCtxCancel := context.WithTimeoutCause(ctx, utils.SyncTimeout, fmt.Errorf("tables restore task creation has not completed in time"))
+		defer tablesRestoreCreationCtxCancel()
+		stdout, stderr, err = utils.ExecWithOptions(tablesRestoreCreationCtx, f.AdminClientConfig(), f.KubeAdminClient().CoreV1(), utils.ExecOptions{
+			Command: []string{
+				"sctool",
+				"restore",
+				fmt.Sprintf("--cluster=%s", targetManagerClusterID),
+				fmt.Sprintf("--location=%s", restoreLocation),
+				fmt.Sprintf("--snapshot-tag=%s", snapshotTag),
+				"--restore-tables",
+			},
+			Namespace:     globalScyllaDBManagerInstancePod.Namespace,
+			PodName:       globalScyllaDBManagerInstancePod.Name,
+			ContainerName: "scylla-manager",
+			CaptureStdout: true,
+			CaptureStderr: true,
+		})
+		o.Expect(err).NotTo(o.HaveOccurred(), stdout, stderr, context.Cause(tablesRestoreCreationCtx))
+
+		_, tablesRestoreTaskID, err := managerClient.TaskSplit(ctx, targetManagerClusterID, strings.TrimSpace(stdout))
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the tables restore task to finish")
+		o.Eventually(verification.VerifyScyllaDBManagerRestoreTaskCompleted).
+			WithContext(ctx).
+			WithTimeout(15*time.Minute).
+			WithPolling(5*time.Second).
+			WithArguments(managerClient, targetManagerClusterID, tablesRestoreTaskID.String()).
+			Should(o.Succeed())
+
+		framework.By("Validating that the data restored from the source cluster backup is available in the target cluster")
+		targetHostsByDC, err = utilsv1alpha1.GetBroadcastRPCAddressesForScyllaDBCluster(ctx, rkcClusterMap, targetSC)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		allTargetHosts = slices.Concat(slices.Collect(maps.Values(targetHostsByDC))...)
+		o.Expect(allTargetHosts).To(o.HaveLen(int(controllerhelpers.GetScyllaDBClusterNodeCount(targetSC))))
+		err = di.SetClientEndpoints(allTargetHosts)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verification.VerifyCQLData(ctx, di)
+	})
+})
+
+func setUpGCSCredentialsForScyllaDBClusterDatacenter(ctx context.Context, coreClient corev1client.CoreV1Interface, sdc *scyllav1alpha1.ScyllaDBClusterDatacenter, namespace string, serviceAccountKey []byte) *scyllav1alpha1.ScyllaDBClusterDatacenter {
+	sdcCopy := sdc.DeepCopy()
```

Comment:
sdc is passed as a pointer, why not modify it directly?

Reply:
at this point for consistency with existing methods for SDC and SC - edited it in all tests